newIDE/app/src/EventsSheet/ParameterFields/ObjectField.js

```diff
@@ -14,6 +14,41 @@ import { Trans } from '@lingui/macro';
 import { nameAndIconContainer } from '../EventsTree/ClassNames';
 import InAppTutorialContext from '../../InAppTutorial/InAppTutorialContext';
 
+const gd: libGDevelop = global.gd;
+
+export const getBehaviorConstraints = (
+  platform: gdPlatform,
+  functionMetadata: gdInstructionMetadata | gdExpressionMetadata,
+  parameterIndex: number
+) => {
+  const behaviorConstraints: Array<{
+    behaviorName: string,
+    behaviorType: string,
+  }> = [];
+  for (
+    let index = parameterIndex + 1;
```

Comment:
Why this `+ 1`?

Reply:
It's looking for behaviors that follow the object.